src/Chapter04/Listing04.28.OutOfScope.cs

```diff
@@ -27,11 +27,15 @@ public static void Main(string[] args)
             {
                 // ...
             }
+            #endregion INCLUDE
+            #if INCLUDE
```

Comment:
I would pick a different symbol name rather than "INCLUDE" it gets confusing seeing it alongside `#region INCLUDE`